src/worker/vm/vm.ts

```diff
@@ -15,7 +16,11 @@ export async function createPluginConfigVM(
     pluginConfig: PluginConfig, // NB! might have team_id = 0
     indexJs: string
 ): Promise<PluginConfigVMReponse> {
-    const transformedCode = transformCode(indexJs, server)
+    const imports = {
+        'node-fetch': { default: fetch },
```

Comment:
Possibly dumb question: by providing `fetch` outright, will nobody be able to access some properties in a way that affects other VMs (e.g. redefine `fetch.isRedirect`)?

Reply:
Definitely not a dumb question. VM2 should wrap everything in proxies. Worth testing it out though. This PR won't really change anything security-wise though, so might be worth tackling after.